manual/mailings/call_for_papers.rst

```diff
@@ -15,19 +16,22 @@ scientific computing in Python, such as:
 - hardware systems designed to support the Python stack
 - applications of scientific computing in education
 
-Papers should be submitted as pull requests by May 30th.  At that point, the 
-two-week open review period will begin.  During this time, members from the 
-program committee and the scientific Python community at large may provide 
+Papers must be submitted as pull requests by May 14th.  At that point, the 
+six-week open review period will begin.  During this time, members from the 
+program committee and the scientific Python community at large will provide 
 feedback, which should be addressed through discussion on GitHub and by 
 updating the pull request.
 
+When submitting a paper to the proceedings, you may be contacted by the
```

Comment:
I think we should decide on something more concrete for how many papers they will be asked to review. Also I think we should be encouraging people to volunteer first and say as a fallback that if we don’t get enough reviews, we will ask authors to review papers.

Reply:
I changed it to "at least one". It's more flexible and we can always *ask* people to review more. If needed, we can pick up the slack.